portal-ecs.yml

```diff
@@ -164,12 +167,25 @@ Parameters:
     Description: OpenGraph title property used for home page
   OpenGraphDescription:
     Type: String
-    Description: OpenGraph descriptio property used for home page and other pages without
-      their own Description
+    Description: OpenGraph description property used for home page and other
+      pages without their own Description
+
   OpenGraphImageURL:
     Type: String
-    Description: OpenGraph image property used for home page ad other pages without their
-      own description
+    Description: OpenGraph image property used for home page ad other pages
+      without their own description
+
+  SubnetIds:
+    Type: String
+    Description: Subnet must be a publicly accessible subnet on the main VPC
+    # QA Value: 'subnet-13a1133d,subnet-2d925713'
+    Default: 'subnet-7fb16326'
+
+  VpcId:
+    Type: String
+    # QA Value: 'vpc-c116d1bb'
+    Description: The VPC to run in.
+    Default: 'vpc-9c9714f9'
```

Comment:
In other places of the template the VpcId is taken from the assosated cluster with: 
`{ 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterVpcId' }`

I think that can be used instead of this parameter.